spec.html

```diff
@@ -47441,28 +47385,28 @@ <h1>Additional Properties of the Global Object</h1>
       <emu-annex id="sec-escape-string">
         <h1>escape ( _string_ )</h1>
         <p>The `escape` function is a property of the global object. It computes a new version of a String value in which certain code units have been replaced by a hexadecimal escape sequence.</p>
-        <p>For those code units being replaced whose value is `0x00FF` or less, a two-digit escape sequence of the form <code>%<var>xx</var></code> is used. For those characters being replaced whose code unit value is greater than `0x00FF`, a four-digit escape sequence of the form <code>%u<var>xxxx</var></code> is used.</p>
+        <p>When replacing a code unit of numeric value less than or equal to 0x00FF, a two-digit escape sequence of the form <code>%<var>xx</var></code> is used. When replacing a code unit of numeric value greater than 0x00FF, a four-digit escape sequence of the form <code>%u<var>xxxx</var></code> is used.</p>
```

Comment:
I don't think it's necessary to say "of numeric value" just for the comparison here. We already test for presence of code units in ranges elsewhere.

Reply:
I'd rather not lose the precise wording here. It also aligns with [The String Type](https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#sec-ecmascript-language-types-string-type) ("_every code unit with a numeric value in the inclusive range 0xD800 to 0xDBFF_") and [UTF16EncodeCodePoint](https://tc39.es/ecma262/multipage/ecmascript-language-source-code.html#sec-utf16encodecodepoint) ("_the code unit whose value is_") (although the latter section lacks "numeric", and I could see going either way on that qualifier).